src/components/tab-panel/examples/tab-panel-content.tsx

```diff
@@ -0,0 +1,101 @@
+import {
+    Component,
+    h,
+    Prop,
+    EventEmitter,
+    Event,
+    State,
+    Watch,
+} from '@stencil/core';
+import { Tab } from '@limetech/lime-elements';
+
+const LOAD_TIME = 1500;
+
+@Component({
+    tag: 'limel-example-tab-panel-content',
+    shadow: true,
+    styleUrl: 'tab-panel-content.scss',
+})
+export class TabPanelContentExample {
+    @Prop()
+    public tab: Tab;
+
+    @Event()
+    private changeTab: EventEmitter<Tab>;
+
+    @State()
+    private votes = 0;
+
+    @State()
+    private loaded = false;
+
+    constructor() {
+        this.vote = this.vote.bind(this);
+    }
+
+    /**
+     * Load the tab contents the first time the tab is activated
+     */
+    @Watch('tab')
+    protected watchTab() {
+        if (!this.loaded && this.tab.active) {
+            this.loadTabContent();
+        }
+    }
+
+    /**
+     * Simulate loading the tab content
+     */
+    private loadTabContent() {
+        setTimeout(() => {
+            this.loaded = true;
+        }, LOAD_TIME);
+    }
+
+    public render() {
+        if (!this.loaded) {
+            return <limel-spinner />;
+        }
+
+        const style = {
+            backgroundColor: this.tab.iconColor,
+            color: 'white',
+        };
+        const text = this.getMovieTitle();
+
+        return (
+            <div class="container">
+                <div class="text">
+                    <limel-icon
+                        badge={true}
+                        size="large"
+                        name={this.tab.icon}
+                        style={style}
+                    />
+                    <p>
+                        {text} as received {this.votes} votes!
+                    </p>
+                </div>
+                <limel-button
+                    outlined={true}
+                    icon="star"
+                    label="Vote"
+                    onClick={this.vote}
+                />
+            </div>
+        );
+    }
+
+    private vote() {
+        this.votes++;
+        const text = this.getMovieTitle();
+        this.changeTab.emit({
+            ...this.tab,
+            text: [text, this.votes].join(' '),
+        });
+    }
+
+    private getMovieTitle() {
+        return this.tab.text.split(' ').shift();
```

Comment:
I don't a point to have this `split(' ').shift()`. What if you have a text `Jojo Rabbit`? It will cut off the `Rabit` part. I suggest to retun `this.tab.text`

Reply:
This is just temporary. We should update the example to use the badge once that is merged :slightly_smiling_face: